src/main/java/frc/trigon/robot/subsystems/sideshooter/SideShooter.java

```diff
@@ -0,0 +1,92 @@
+package frc.trigon.robot.subsystems.sideshooter;
+
+import com.ctre.phoenix6.controls.VoltageOut;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.trigon.robot.utilities.Conversions;
+
+public class SideShooter extends SubsystemBase {
+    private final static SideShooter INSTANCE = new SideShooter();
+    private final TalonFX shootingMotor = SideShooterConstants.SHOOTING_MOTOR;
+    private final CANSparkMax angleMotor = SideShooterConstants.ANGLE_MOTOR;
+    private final CANcoder angleEncoder = SideShooterConstants.ANGLE_ENCODER;
+
+    private final VoltageOut shootingVoltageRequest = new VoltageOut(0, SideShooterConstants.FOC_ENABLED, false);
+
+    private TrapezoidProfile
+            angleMotorProfile = null;
+
+    private double lastAngleMotorProfileGenerationTime;
+
+    public static SideShooter getInstance() {
+        return INSTANCE;
+    }
+
+    private SideShooter() {
+    }
+
+    boolean ifGotToAngle(double atAngle){
+        if (Math.abs(atAngle - getAnglePosition().getDegrees()) <= 1){
+            return true;
+        };
+        return false;
+    }
```

Comment:
rename this to `atAngle`, make it accept a Rotation2d as a param, and fix the body of the function. Intellij should suggest a better way.